content/working_with/console/widgets/snapshots.md

```diff
@@ -30,7 +39,49 @@ The widget also exposes the following operations by the buttons on the top right
 * **Create** - Create a new snapshot on the current tenant
 * **Upload** - Upload a snapshot to the current tenant
 
-![snapshots-list]( /images/ui/widgets/snapshots-list.png )
 
-#### Widget Settings
+#### Creating a Snapshot
+
+The snapshots creation process captures data in the entire Cloudify Manager, not just that of a specific tenant. However, the snapshot is created in the context of the current tenant, and therefore must be restored from it.
+
+1. Click **Create** button above the Snapshots table.
+2. Specify a unique ID for the snapshot and click **Create** button.   
+   It is good practice to use a name that will help you to easily identify the snapshot later.
+
+The creation process begins. If there are active executions when you attempt to create the snapshot, the process waits until the executions are complete before creating the snapshot. You can see the status of executions in the Deployment executions widget.
+
+The snapshot is saved as a ZIP file and appears in the Snapshots table, together with details of its creator, creation date and time, and current status.
+
+
+#### Restoring a Snapshot
+
+If you are restoring a snapshot from a Cloudify Manager instance prior to version 4.0, refer to the [Restoring Snapshots of Legacy Cloudify Manager Instances]({{< relref "working_with/console/pages/admin-operations-page.md" >}}#restoring-snapshots-of-legacy-cloudify-manager-instances) section below.
+
+If you restore a snapshot to a Cloudify Manager instance that already contains data, that data is overwritten. To prevent inadvertent overwriting of existing data, you must explicity state that you want to force data overwrite.
+
+1. Click **Upload** button in the widget.
+2. Either enter the URL of the snapshot or select the snapshot file from your file repository.
+3. Enter the Snapshot name.
+4. Click **Upload** button and see that snapshot was uploaded and is available in Snapshots table.
+5. Click Restore icon ![Restore icon]( /images/ui/icons/restore-icon.png ) on the far right of newly uploaded snapshot's row
+   * If your snapshot is from a Cloudify Manager instance that was created earlier than version 4.0, see [Restoring Snapshots of Legacy Cloudify Manager Instances]({{< relref "working_with/console/pages/admin-operations-page.md" >}}#restoring-snapshots-of-legacy-cloudify-manager-instances).
```

Comment:
Again, please remove this sentence completely. we no longer support such an upgrade.